frontends/web/src/routes/device/bitbox02/bitbox02.module.css

```diff
@@ -1,3 +1,17 @@
-.agreements {
-    margin-bottom: var(--spacing-default);
+/* TODO: ideally fontsize of those labels would just use the default from View component */
```

Comment:
`device/bitbox02.tsx` didn't have a css file somehow all styles were in `steps.module.css` that is mostly not needed anymore after moving to View component and deleted in this PR.

The styles are about the checkboxes where the users agrees to understand. Added a comment to look into removing those.